Loader/SymfonyFixturesLoader.php

```diff
@@ -65,4 +83,70 @@ protected function createFixture($class)
 
         return $this->loadedFixtures[$class];
     }
+
+    /**
+     * Returns the array of data fixtures to execute.
+     *
+     * @param string[] $groups
+     *
+     * @return FixtureInterface[]
+     */
+    public function getFixtures(array $groups = [])
+    {
+        $fixtures = parent::getFixtures();
+
+        if (empty($groups)) {
+            return $fixtures;
+        }
+
+        $filteredFixtures = [];
+        foreach ($fixtures as $fixture) {
+            foreach ($groups as $group) {
+                $fixtureClass = get_class($fixture);
+                if (isset($this->groupsFixtureMapping[$group][$fixtureClass])) {
+                    $filteredFixtures[$fixtureClass] = $fixture;
+                    continue 2;
+                }
+            }
+        }
+
+        $loadedFixtureClasses = array_keys($filteredFixtures);
+        foreach ($filteredFixtures as $fixture) {
+            $this->validateDependencies($loadedFixtureClasses, $fixture);
```

Comment:
You should just send the array `$filteredFixtures` as the first argument here, instead of the array keys of that array.
Then later you can use `isset` or `array_key_exists` instead of the much slower `in_array`